src/split-panel/styles.scss

```diff
@@ -100,16 +97,19 @@ $app-layout-drawer-width: calc(#{awsui.$space-layout-toggle-diameter} + 2 * #{aw
 }
 
 .position-side {
-  border-top: none;
+  border-block-start: none;
   box-shadow: awsui.$shadow-split-side;
-  height: 100%;
+  block-size: 100%;
 }
 
 .slider {
-  padding: 0;
+  padding-block: 0;
+  padding-inline: 0;
   cursor: ns-resize;
-  margin: 2px 0 0 0;
-  height: $slider-width;
+  margin-block-start: 2px;
```

Comment:
Can we use the same shorthand syntax like `margin-block: 2px 0` and `margin-inline: 0 0`?

Reply:
Yes!